src/discretization/1_discretization.cpp

```diff
@@ -0,0 +1,65 @@
+
+#include "0_staggered_grid.h"
+
+// Contructor
+Discretization::Discretization(std::array<int,2> nCells, std::array<double,2> meshWidth) :
+    StaggeredGrid(nCells, meshWidth)
+{
+}
+
+
+//! compute the 2nd derivative ∂^2 u / ∂x^2 at right edge
+double computeD2uDx2(int i, int j) const
```

Comment:
You have to add the namespace to all method calls like this
```suggestion
double Discretization::computeD2uDx2(int i, int j) const
```

Reply:
done